<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1350</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1350</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QStatusBar{
border: none;
}</string>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1351</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
background:rgba(0,0,0,0.8);
}
QFrame{

border-radius:none;

}

QFrame:hover{
background:rgba(0,0,0,0.9);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
color:silver;
background:transparent;</string>
     </property>
     <property name="text">
      <string>Asosiy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_close">
     <property name="geometry">
      <rect>
       <x>1310</x>
       <y>0</y>
       <width>41</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background:rgba(0, 0, 0,0.15);
color:silver;

border-radius:none;
}


QPushButton:hover{
background:red;
color:white
}</string>
     </property>
     <property name="text">
      <string>âœ•</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_minimize">
     <property name="geometry">
      <rect>
       <x>1270</x>
       <y>0</y>
       <width>41</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background:rgba(0, 0, 0,0.15);
border-radius:none;
color: white;
}


QPushButton:hover{
background:rgba(0, 0, 0,0.6)
}</string>
     </property>
     <property name="text">
      <string>_</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>371</x>
      <y>40</y>
      <width>981</width>
      <height>763</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background-image: url(:/images/img/back_1.jpg);
}
QWidget{
  font-size: 20px;
  font-weight: 600;
  color: transparent;
  background-clip: text;
  border: none;
  -webkit-background-clip: text;
}
QTabWidget::tab-bar {

 }
QTabBar::tab{
 width: 225px;
  background: rgba(0,0,0,0.6);
  color: white;
  padding: 10px;
}


 QTabBar::tab:selected {
 
background: rgba(195,63,11,0.6);
 }
 QTabBar::tab:hover {
   background: rgba(82,120,21, 0.7);
 }
QFrame{
	background: rgba(0,0,0,0.6);
}
QFrame:hover{
	background: rgba(0,0,0,0.7);
}
QPushButton{
background: rgba(255,255,250,0.9);
}
</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab_ovqat">
     <attribute name="title">
      <string>Ovqatlar</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1301</width>
        <height>721</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>121</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_salat">
     <attribute name="title">
      <string>Salatlar</string>
     </attribute>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1301</width>
        <height>721</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_shirinlik">
     <attribute name="title">
      <string>Shirinliklar</string>
     </attribute>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1301</width>
        <height>721</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_ichimlik">
     <attribute name="title">
      <string>Ichimliklar</string>
     </attribute>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>-40</x>
        <y>0</y>
        <width>1341</width>
        <height>721</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>373</width>
      <height>761</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background-image: url(:/images/img/list_back.jpg);
}
QWidget{
  font-size: 14px;
  font-weight: 500;
  color: transparent;
  background-clip: text;
  border: none;
  -webkit-background-clip: text;
}
QTabWidget::tab-bar {

 }
QTabBar::tab{
 width: 165px;
  background: rgba(0,0,0,0.6);
  color: white;
  padding: 10px;
}


 QTabBar::tab:selected {
 
background: rgba(195,63,11,0.6);
 }
 QTabBar::tab:hover {
   background: rgba(82,120,21, 0.7);
 }
QFrame{
	background: rgba(0,0,0,0.6);
  filter: blur(8px);
  -webkit-filter: blur(8px);
}
QFrame:hover{
	background: rgba(0,0,0,0.7);
}
QLabel{
color: white;
}</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_xarid">
     <attribute name="title">
      <string>Xarid</string>
     </attribute>
     <widget class="QFrame" name="frame_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>371</width>
        <height>721</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QScrollArea" name="scrollArea">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>5</y>
         <width>361</width>
         <height>710</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background:rgba(0, 0, 0,0.15);
border-radius:none;
color: white;
}

QPushButton:hover{
background:rgba(0, 0, 0,0.6)
}</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>361</width>
          <height>710</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="verticalSpacing">
            <number>10</number>
           </property>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_profil">
     <attribute name="title">
      <string>Profil</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>375</width>
        <height>731</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_buyurtma">
       <attribute name="title">
        <string>Buyurtmalar</string>
       </attribute>
       <widget class="QFrame" name="frame_8">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>371</width>
          <height>691</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_setting">
       <attribute name="title">
        <string>Sozlash</string>
       </attribute>
       <widget class="QFrame" name="frame_9">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>371</width>
          <height>691</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <zorder>tabWidget_2</zorder>
   <zorder>frame_2</zorder>
   <zorder>tabWidget</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
